examples/angular/testDemoAngulaApp.js

```diff
@@ -0,0 +1,48 @@
+describe('Angular Demo Example', () => {
+  const firstInput = 'input[ng-model=\'first\']';
```

Comment:
string should be `'input[ng-model="first"]'`

Reply:
IMO, both string will work because
```HTML
"<li class=\"out\">" // escape the quotes to be ignored with \.
"<li class='out'>" // use different quotes on the class name.
'<li class="out">' // use different quotes on the piece of html.
```
All above string will fetch dom element

Ref:
![image](https://user-images.githubusercontent.com/8705386/117121731-a4775500-adb2-11eb-942b-172f6e36a294.png)

Reply:
Yes, but by using double-quotes you avoid the need for escaping the single quotes, which improves readability, and also standard html code uses double quotes, so there's no benefit of using single quotes here.

Reply:
Alright, will change it to double quotes